docs-src/app-dev-context/custom-payload-conversion.md

```diff
@@ -0,0 +1,19 @@
+---
+id: custom-payload-conversion
+title: How to use custom payload conversion
+description: Create your custom `PayloadConverter` and set it on a `DataConverter` in your Client options.
+sidebar_label: Custom payload conversion
+tags:
+  - guide-context
+---
+
+Most SDKs provide a [`PayloadConverter`](/concepts/what-is-a-payload-converter) that can be customized to convert values in custom data types to bytes and back.
+
+Creating a custom Payload Converter is optional.
```

Comment:
This sentence doesn't flow right. Maybe "Creation of ..."?

Reply:
Changed to "Implementing custom Payload conversion is optional."